InnoGotchi.MVC/InnoGotchi.MVC/Middlewares/AddingAuthenticationHeaderMiddleware.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace InnoGotchi.MVC.Middlewares
+{
+    public class AddingAuthenticationHeaderMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public AddingAuthenticationHeaderMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            var token = context.Request.Cookies["jwt"];
```

Comment:
Is there a better way? Can you send the token in the header immediately and delete this middleware?

Reply:
I didn't find a better way, but I can call MVC project by jquery and then pass token in header. I don't know another way how to add header to request. Should I use jquery to all my calls to mvc?